### PR TITLE
fix: fix copy button not hidden after copy

### PR DESCRIPTION
**Pull request by artemmufazalov**
Copy button didn't hide after copy. Made it be shown only on hover

Test stand: https://nda.ya.ru/t/01pCgQBZ763DPX

### CODE REVIEW

File: src/containers/App/App.scss [hunk 1]
```diff
@@ -119,8 +119,7 @@ body,
 .data-table__row,
 .ydb-virtual-table__row,
 .ydb-tree-view__item {
-    &:hover,
-    &:focus-within {
+    &:hover {
```
Comment (Raubzeug) [inline src/containers/App/App.scss:122]: let's add `focus-visible` here, to make the button visible if iterating with keyboard.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/App/App.scss (hunk 1 above)
Comment (artemmufazalov) [reply src/containers/App/App.scss:122]: Elements with `visibility:hidden` cannot be selected focused with tab, so it doesn't have any effect

Comment (Raubzeug) [reply src/containers/App/App.scss:122]: 100%! What do you think if we hide them with `opacity=0`?

Comment (artemmufazalov) [reply src/containers/App/App.scss:122]: `opacity=0` works just fine, updated PR

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**